### DISCUSSION TITLE
I have an entity like this that I want to bind to. I can change the background color and font color, as you can see, and my entity header can be dynamically added. Can you help me take a look

### DISCUSSION THREAD

**Discussion by wosiyourfather** [AvaloniaUI/Avalonia#12629] [Q&A]
I have an entity like this that I want to bind to. I can change the background color and font color, as you can see, and my entity header can be dynamically added. Can you help me take a look
[demo.zip](https://github.com/AvaloniaUI/Avalonia/files/12406815/demo.zip)

**Comment by wosiyourfather** [comment]
I want to bind ”Datacls“ data into it

**Comment by thevortexcloud** [comment]
I am not clear what you are trying to do by reading the code. But there are examples here on how to use the tree datagrid:

https://github.com/AvaloniaUI/Avalonia.Controls.TreeDataGrid/blob/master/docs/get-started-hierarchical.md
